net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/sites-enabled-inner-tunnel

```diff
@@ -242,9 +251,11 @@ authenticate {
         #  authentication server, and knows what to do with authentication.
         #  LDAP servers do not.
         #
+{%   if helpers.exists('OPNsense.freeradius.ldap.innertunnel') and OPNsense.freeradius.general.ldap.innertunnel == '1' %}
 #       Auth-Type LDAP {
```

Comment:
Sorry for asking, correct me if I'm wrong. But shouldn't these lines look like this?:
```
Auth-Type LDAP {
               ldap
}
```
Not like this:

```
# Auth-Type LDAP {
#               ldap
# }
```